frontend/packages/dev-console/integration-tests/support/step-definitions/common/addFlow.ts

```diff
@@ -103,14 +103,15 @@ When('user enters run command for {string} as {string}', (envKey: string, value:
 });
 
 Then(
-  'user is able to navigate to Build #1 for deployment {string} and see environment variable {string} in Environment tab of details page',
-  (name: string, env: string) => {
+  'user is able to navigate to Build {string} for deployment {string} and see environment variable {string} with value {string} in Environment tab of details page',
+  (build: string, name: string, envKey: string, envVal: string) => {
     topologyPage.clickOnNode(name);
     topologySidePane.selectTab('Resources');
-    topologySidePane.selectResource(resources.Builds, 'aut-addflow-git', `${name}-1`);
+    topologySidePane.selectResource(resources.Builds, 'aut-addflow-git', build);
     app.waitForLoad();
     detailsPage.selectTab('Environment');
     app.waitForLoad();
-    cy.get(`input[data-test="pairs-list-name"][value="${env}"]`).should('exist');
+    cy.get(`input[data-test="pairs-list-name"][value="${envKey}"]`).should('exist');
+    cy.get(`input[data-test="pairs-list-value"][value="${envVal}"]`).should('exist');
```

Comment:
Nit. Can we also check that it only exists once? Wdyt?

(Again, only add this comment because you need to touch the PR anyway because saving doesn't work when start command is not defined.)

```suggestion
    cy.get(`input[data-test="pairs-list-name"][value="${envKey}"]`).should('have.length', 1)
    cy.get(`input[data-test="pairs-list-value"][value="${envVal}"]`).should('have.length', 1)
```

Reply:
Updated.